meta-lmp-bsp/dynamic-layers/freescale-layer/recipes-bsp/imx-atf/imx-atf_%.bbappend

```diff
@@ -6,7 +6,10 @@ RPROVIDES_${PN} += "virtual/trusted-firmware-a"
 SRC_URI_append_toradex = " \
     file://0001-Revert-Add-NXP-s-SoCs-partition-reboot-support.patch \
 "
-
```

Comment:
@ricardosalveti forgot to address you comment about putting SRC_URI_append before SRC_URI_append_toradex